README.md

```diff
@@ -12,7 +12,7 @@ The model is based on the [Show and Tell Image Caption Generator Model](https://
 | Vision | Image Caption Generator | General | TensorFlow | [COCO](http://cocodataset.org/#home) | Images | 
 
 ## References
-* _O. Vinyals, A. Toshev, S. Bengio, D. Erhan._, ["Show and Tell: Lessons learned from the 2015 MSCOCO Image Captioning Challenge"](https://arxiv.org/abs/1609.06647), IEEE transactions on Pattern Analysis and Machine Intelligence, 2016.
+* _O. Vinyals, A. Toshev, S. Bengio, D. Erhan._, ["Show and Tell: Lessons learned from the 2015 MSCOCO Image Captioning Challenge"](https://ieeexplore.ieee.org/document/7505636), IEEE transactions on Pattern Analysis and Machine Intelligence, 2017.
```

Comment:
https://doi.org/10.1109/TPAMI.2016.2587640 might be better, in case the publisher moves their documents to a different URL